docs/ocm-control-plane-walkthrough.md

```diff
@@ -268,55 +275,55 @@ EOF
 
 Once this is done, the Kuadrant multi-cluster gateway controller will pick up that a HTTPRoute has been attached to the Gateway it is managing from the hub and it will setup a DNS record to start bringing traffic to that gateway for the host defined in that listener.
 
-You should now see a DNSRecord resource in the hub cluster:
+You should now see a DNSRecord resource in the hub cluster. In `T1`, run:
 
-```
+```bash
 kubectl get dnsrecord -A
 NAMESPACE                NAME                 READY
 multi-cluster-gateways   sub.replace.this   True
 
 ```
 
-you should also be able to see there is only 1 endpoint added which corresponds to address assigned to the gateway where the HTTPRoute was created.
+You should also be able to see there is only 1 endpoint added which corresponds to address assigned to the gateway where the HTTPRoute was created. In `T1`, run:
 
-```
+```bash
 kubectl get dnsrecord -n multi-cluster-gateways -o=yaml
 ```
 
 Give DNS a minute or two to update. You should then be able to execute
 
-```
+```bash
 dig sub.replace.this 
```

Comment:
To make things a little smoother, what about introducing a env var early on in the walkthrough for the hostname?

Reply:
Had a similar thought when I last went through this on Friday